.kokoro/test.sh

```diff
@@ -1,48 +0,0 @@
-#!/bin/bash
-
-# Copyright 2018 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -eo pipefail
-
-export NPM_CONFIG_PREFIX=/home/node/.npm-global
-
-cd $(dirname $0)/..
-
-npm install
-# If tests are running against master, configure Build Cop
-# to open issues on failures:
-if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"continuous"* ]]; then
-  export MOCHA_REPORTER_OUTPUT=test_output_sponge_log.xml
-  export MOCHA_REPORTER=xunit
```

Comment:
I would like to keep this until we have a plan to provide something like build cop for our actions; the rest looks pretty slick👌